src/settings.cpp

```diff
@@ -0,0 +1,73 @@
+#include "settings.h"
+
+#include <ArduinoJson.h>
+#include <SdFat.h>
+
+#include "filesystem.h"
+#include "types.h"
+static char SETTINGS_FILE[FILE_NAME_SIZE] = "settings.json";
+extern SdFs sd;
+
+enum SettingsConstants {
+    LOG_ALTITUDE_THRESHOLD_DEFAULT = 100, // meters
+    BARO_ALTITUDE_TOLERANCE_DEFAULT = 5,  // meters
+    GPS_ALTITUDE_TOLERANCE_DEFAULT = 5,   // meters
+};
```

Comment:
is there a reason this is an enum. wouldnt a constexpr (maybe in a namespace if you really want your ::s) be a better choice (esp bc these can be floats)